### ISSUE TITLE
[JSON] SecurityLogClient sends messages as tuples

### ISSUE THREAD

**Issue by chaen** [DIRACGrid/DIRAC#4324]
```
============================================= Encoding tuples =============================================
  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/diracos/usr/lib64/python2.7/threading.py", line 777, in __bootstrap
    self.__bootstrap_inner()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/diracos/usr/lib64/python2.7/threading.py", line 804, in __bootstrap_inner
    self.run()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/diracos/usr/lib64/python2.7/threading.py", line 757, in run
    self.__target(*self.__args, **self.__kwargs)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 118, in __executorThread
    timeToNext = self.executeNextTask()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 134, in executeNextTask
    self.__executeTask( taskId )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 187, in __executeTask
    task[ 'func' ]( *task[ 'args' ] )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/FrameworkSystem/Client/SecurityLogClient.py", line 52, in __sendData
    result = rpcClient.logActionBundle(msgsToSend)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RPCClient.py", line 43, in __call__
    return self.__doRPCFunc( self.__remoteFuncName, args )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RPCClient.py", line 90, in __doRPC
    return self.__innerRPCClient.executeRPC( sFunctionName, args )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/InnerRPCClient.py", line 64, in executeRPC
    retVal = transport.sendData( S_OK( list(args) ) )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py", line 162, in sendData
    sCodedData = DEncode.encode(uData)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 393, in encode
    g_dEncodeFunctions[type(uObject)](uObject, eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 361, in encodeDict
    g_dEncodeFunctions[type(dValue[key])](dValue[key], eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 305, in encodeList
    g_dEncodeFunctions[type(uObject)](uObject, eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 305, in encodeList
    g_dEncodeFunctions[type(uObject)](uObject, eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 328, in encodeTuple
    printDebugCallstack('Encoding tuples')

Calling frame: ('/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py', 162)
With arguments  {'uObject': {'OK': True,
             'Value': [[(datetime.datetime(2019, 10, 24, 14, 20, 6, 539365),
                         True,
                         '::1',
                         45832,
                         '[localhost:hosts](/C=ch/O=DIRAC/OU=DIRAC CI/CN=diracci6-02.cern.ch/emailAddress=lhcb-dirac-ci@cern.ch)',
                         'localhost',
                    9149,
                         'StorageManagement/StorageManager',
                         'RPC/getTasksWithStatus'),
                        (datetime.datetime(2019, 10, 24, 14, 20, 6, 703300),
                         True,
                         '::1',
                         45838,
                         '[localhost:hosts](/C=ch/O=DIRAC/OU=DIRAC CI/CN=diracci6-02.cern.ch/emailAddress=lhcb-dirac-ci@cern.ch)',
                         'localhost',
                         9149,
                         'StorageManagement/StorageManager',
                         'RPC/removeUnlinkedReplicas'),
                        (datetime.datetime(2019, 10, 24, 14, 20, 6, 739262),
                         True,
                         '::1',
                         45840,
                         '[localhost:hosts](/C=ch/O=DIRAC/OU=DIRAC CI/CN=diracci6-02.cern.ch/emailAddress=lhcb-dirac-ci@cern.ch)',
                         'localhost',
                         9149,
                         'StorageManagement/StorageManager',
                         'RPC/setOldTasksAsFailed'),
                        (datetime.datetime(2019, 10, 24, 14, 20, 11, 46608),
                         True,
                         '::1',
                         45866,
                         '[localhost:hosts](/C=ch/O=DIRAC/OU=DIRAC CI/CN=diracci6-02.cern.ch/emailAddress=lhcb-dirac-ci@cern.ch)',
                         'localhost',
                         9149,
                         'StorageManagement/StorageManager',
                         'RPC/getCacheReplicas'),
                        (datetime.datetime(2019, 10, 24, 14, 20, 14, 922641),
                         True,
                         '::1',
                         45886,
                         '[localhost:hosts](/C=ch/O=DIRAC/OU=DIRAC CI/CN=diracci6-02.cern.ch/emailAddress=lhcb-dirac-ci@cern.ch)',
                         'localhost',
                         9149,
                         'StorageManagement/StorageManager',
                         'RPC/getCacheReplicas')]]}}
====================================================================================================
```

**Comment by chaen** [comment]
In https://github.com/DIRACGrid/DIRAC/pull/4366

**Comment by chrisburr** [comment]
I think this can be closed now #4410 has been merged?
